src/components/views/read/cvForm/FormLayout.tsx

```diff
@@ -0,0 +1,327 @@
+        <div
+          style={{
+            alignSelf: 'stretch',
+            flex: '1',
+            display: 'flex',
+            flexDirection: 'column',
+            padding: '16px 0px 0px',
+            alignItems: 'flex-start',
+            justifyContent: 'flex-start',
+            gap: '8px',
+            textAlign: 'right',
+          }}
+        >
+          <Box
+            style={{ alignSelf: 'stretch', flex: '1' }}
+            sx={{ width: '100%' }}
+          >
+            <Stepper activeStep={activeStep} alternativeLabel>
+              {steps.map((label, index) => (
+                <Step key={index}>
+                  <StepLabel>{label}</StepLabel>
+                </Step>
+              ))}
+            </Stepper>
+          </Box>
+        </div>
+      </div>
+      <div
+        style={{
+          position: 'absolute',
+          top: '290px',
+          left: '450px',
+          width: '1001px',
+          height: '463px',
+          textAlign: 'center',
+          color: 'rgba(0, 0, 0, 0.87)',
+        }}
+      >
+        {/* Modify the below sizes to change the sizes of the white box on the screen */}
+        <div
+          style={{
+            position: 'absolute',
+            top: '0px',
+            left: '0px',
+            borderRadius: '4px',
+            backgroundColor: '#fff',
+            boxShadow:
+              '0px 1px 1px rgba(0, 0, 0, 0.14), 0px 2px 1px rgba(0, 0, 0, 0.12), 0px 1px 3px rgba(0, 0, 0, 0.2)',
+            width: '1001px',
+            height: '550px',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+            justifyContent: 'flex-start',
+          }}
+        >
+          {/* <div
+            style={{
+              alignSelf: 'stretch',
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              // justifyContent: 'right',
+              fontSize: '20px',
+            }}
+          > */}
+          {/* <div
+            style={{
+              width: '283px',
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'flex-start',
+              gap: '16px',
+            }}
+          >
+            <div
+              style={{
+                flex: '1',
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'flex-start',
+                justifyContent: 'flex-start',
+                gap: '2px',
+              }}
+            > */}
+          <div
+            style={{
+              alignSelf: 'stretch',
+              flexDirection: 'row',
+              alignItems: 'right',
+              justifyContent: 'right',
+              fontSize: '20px',
+            }}
+          >
+            {activeStep === 0 && (
+              <>
+                <div
+                  style={{
+                    alignSelf: 'stretch',
```

Comment:
What is this ? It is similar in structure to the steps "switch" you have inside the `PlayerView` but only to show a debug text it looks like, why not put that text where you instantiate the steps components ?